python/delta_sharing/tests/test_reader.py

```diff
@@ -237,3 +237,12 @@ def list_files_in_table(
     expected = reader.to_pandas().iloc[0:0]
 
     pd.testing.assert_frame_equal(pdf, expected)
+
+
+@pytest.mark.skipif(not ENABLE_INTEGRATION, reason=SKIP_MESSAGE)
+def test_load_gzip_table(rest_client: DataSharingRestClient):
```

Comment:
It's better to add a test in `https://github.com/delta-io/delta-sharing/blob/main/python/delta_sharing/tests/test_delta_sharing.py#L112` instead.

Reply:
Sure, I thought I'd put it here because it was more related to the reader.